ui/tests/acceptance/enterprise-namespaces-test.js

```diff
@@ -47,7 +47,7 @@ module('Acceptance | Enterprise | namespaces', function (hooks) {
       // check that the single namespace "beep" or "boop" not "beep/boop" shows in the toggle display
```

Comment:
based on your comment below, it sounds like we should update this comment?

Reply:
yes - thank you so much for catching this! updated ✔️